python/ray/tests/test_advanced.py

```diff
@@ -172,15 +172,20 @@ def get_path2():
     reason="Only tested in client/profiling build.")
 def test_profiling_api(ray_start_2_cpus):
```

Comment:
@ericl  I think this test case is polluted by the previous ones since if you run this one separately it'll fail (before this PR), but if you run the suite it's ok. Somehow it fails this PR. I still can't figure out what's happening there and it looks like if it's client mode, somehow internal kv is not cleared across tests.

Could you double confirm this update is OK? And do you have know knowledge about this?

Reply:
Is this still failing on CI? `RAY_CLIENT_MODE=1 RAY_PROFILING=1 pytest -vs ray/tests/test_advanced.py -k test_profiling_api` passes on my laptop, on latest master.

Reply:
Yes, it still fails. I just built on master. I'm using Linux, could it cause any difference?